src/secret_sharing_schemes/mod.rs

```diff
@@ -0,0 +1,2 @@
+pub mod replicated_secret_sharing;
```

Comment:
I don't think we need to make the whole module visible, just re-exporting secret sharing structs is enough